### PR TITLE
Fix: Helmet not working properly #4

### PR DESCRIPTION
**Pull request by dronsfield**
use Head component inside GenericPage and clean up Head nested property errors

### CODE REVIEW

File: src/components/Head/index.js [hunk 1]
```diff
@@ -1,13 +1,16 @@
 import Helmet from "react-helmet";
 import PropTypes from "prop-types";
 import React from "react";
+import { path } from "ramda"
```
Comment (mcclowes) [inline src/components/Head/index.js:4]:
```suggestion
import * as R from "ramda"
```

File: src/components/Head/index.js [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
 
 // --------------------------------------------------
 
 import { theme, } from "../../styles";
 
 
 const Head = ( { site, page, } ) => {
+	const image = path(['image', 'url'])(page) || path(['homeImage', 'url'])(site)
```
Comment (mcclowes) [inline src/components/Head/index.js:12]:
```suggestion
	const image = R.path(['image', 'url'])(page) || R.path(['homeImage', 'url'])(site)
```

Comment (mcclowes) [reply src/components/Head/index.js:12]: I can see myself getting minorly confused at some future date without the `R.` lol

**Review by mcclowes — CHANGES_REQUESTED** [review]

**Comment by mcclowes** [comment]
Any thoughts on this @dronsfield?

**Comment by dronsfield** [comment]
yeah I think it should be changed

**Pull request merged**